waspc/data/Generator/templates/server/src/crud/_operations.ts

```diff
@@ -8,7 +8,7 @@ import type {
 {=# isAuthEnabled =}
 import { throwInvalidCredentialsError } from 'wasp/auth/utils'
 {=/ isAuthEnabled =}
-import type { GetAllQuery, GetQuery, CreateAction, UpdateAction, DeleteAction } from "{= crudTypesImportPath =}";
+import type { {= crud.name =} } from "wasp/server/crud";
```

Comment:
Fixing the CRUD bug in `new-sdk` branch (should be resolved after we merge the `wasp/client/crud` API changes)